### DISCUSSION TITLE
Build Renovate and add to releases

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#24803] [Suggest an Idea]
### Describe the proposed change(s).

Once a GitHub release is done, let's "build" Renovate and it to each release. Perhaps can start with the JS version (using `ncc`) but `pkg` is also an option if we only need one platform

**Comment by HonkingGoose** [comment]
> Once a GitHub release is done, let's "build" Renovate and it to each release. Perhaps can start with the JS version (using `ncc`) but `pkg` is also an option if we only need one platform

Do you mean `@vercel/ncc` [^vercel/ncc]?

I'm not sure what "build Renovate and add it to each release means" here. Do you intend for people to be able to "download Renovate" from the GitHub releases page?

[^vercel/ncc]: https://github.com/vercel/ncc

**Comment by viceice** [comment]
It's to simplify distribution with all deps, once this is done, we can use that to install renovate to our buildpack docker images instead of making some long running yarn installs
